RFC-0010-mixed-case-identifier-support.md

```diff
@@ -0,0 +1,201 @@
+# **RFC-0010 for Presto**
+
+## Mixed case identifiers
+
+Proposers
+
+* Reetika Agrawal
+
+## Summary
+
+Improve Presto's identifier (schema, table & column names) handling to align with SQL standards, ensuring better interoperability with case-sensitive
+and case-normalizing databases while minimizing SPI-breaking changes.
+
+## Background
+
+Presto treats all identifiers as case-insensitive, normalizing them (typically to lowercase). This creates issues when
+querying databases that are case-sensitive (e.g., MySQL, PostgreSQL) or case-normalizing to uppercase (e.g., Oracle,
+DB2). Without a standard approach, identifiers might not match the actual names in the underlying data sources, leading
+to unexpected query failures or incorrect results. Additionally, inconsistent handling of delimited and non-delimited
+identifiers across different connectors further complicates cross-engine compatibility.
+
+The goal here is to improve interoperability with storage engines by aligning identifier handling with SQL standards
```

Comment:
Goals can be listed as bullet points:

* Improve interoperability with storage engines i.e standardizing identifier handling (SQL Standard ref: [ANSI?](https://www.ibm.com/docs/en/netezza?topic=md-handle-sql-identifiers#c_dbuser_handle_sql_identifiers__title__2)).
* Backwards compatibility with existing system. Minimal or no breaking changes to SPI.

*